runtime/include/chplcgfns.h

```diff
@@ -39,6 +39,7 @@
 /* defined in chpl_compilation_config.c: */
 extern const char* chpl_compileCommand;
 extern const char* chpl_compileVersion;
+extern const char* chpl_compileDirectory;
```

Comment:
Would you add back in the blank line that used to precede CHPL_HOME? It was nice to have that separator between the compiler related strings and the environment (config) related strings.